features/step_definitions/cabinet_ministers_steps.rb

```diff
@@ -16,8 +16,11 @@
 
 Given(/^there are multiple Cabinet minister roles$/) do
   organisation = create(:organisation)
-  create(:ministerial_role, name: "Role 1", cabinet_member: true, organisations: [organisation], seniority: 0)
-  create(:ministerial_role, name: "Role 2", cabinet_member: true, organisations: [organisation], seniority: 1)
+  person = create(:person, forename: "Tony")
+  minister_role1 = create(:ministerial_role, name: "Role 1", cabinet_member: true, organisations: [organisation], seniority: 0)
+  minister_role2 = create(:ministerial_role, name: "Role 2", cabinet_member: true, organisations: [organisation], seniority: 1)
+  create(:role_appointment, role: minister_role1, person:)
+  create(:role_appointment, role: minister_role2, person:)
```

Comment:
Can we add `with appointments` or similar to the name of this method to be a bit more explicit?